src/c++/perf_analyzer/docs/llm.md

```diff
@@ -31,9 +31,64 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 The following guide shows the reader how to use Triton
 [Perf Analyzer](https://github.com/triton-inference-server/client/tree/main/src/c%2B%2B/perf_analyzer)
 to measure and characterize the performance behaviors of Large Language Models
-(LLMs) using Triton with [vLLM](https://github.com/vllm-project/vllm).
+(LLMs) using Triton with [TensorRT-LLM](https://github.com/NVIDIA/TensorRT-LLM)
+and [vLLM](https://github.com/vllm-project/vllm).
 
-### Setup: Download and configure Triton vLLM Backend
+## Setup: Download and configure Triton Server and Client environment
+
+### Using TensorRT-LLM
+
+1. Follow [step 1](https://github.com/triton-inference-server/tutorials/blob/main/Popular_Models_Guide/Llama2/trtllm_guide.md#installation)
+of the Installation section.
+
+    This guide will replace the llama model for gpt since it is already
+    included in the tensorrtllm_backend repository release.
+
+    Currently, the most recent release is 0.6.1.
+
+2. Launch the Triton docker container with the TensorRT-LLM backend.
+This will require mounting the repo from step 1 into the docker container
+and any models you plan to serve.
+
+    For the tensorrtllm_backend repository, you need the following two directories mounted:
+- backend: .../tensorrtllm_backend/:/tensorrtllm_backend
+- engine: .../tensorrtllm_backend/tensorrt_llm/examples/gpt/gpt_outputs:/engines
+
+```
+docker run --rm -it --net host --shm-size=2g \
+    --ulimit memlock=-1 --ulimit stack=67108864 --gpus all \
+    -v /path/to/tensorrtllm_backend:/tensorrtllm_backend \
+    -v /path/to/model/repo:/repo \
+    -v /path/to/engines:/engines \
+    nvcr.io/nvidia/tritonserver:23.10-trtllm-python-py3 \
+    bash
+```
+
+3. Create the [engine](https://github.com/triton-inference-server/tutorials/blob/main/Popular_Models_Guide/Llama2/trtllm_guide.md#create-engines-for-each-model-skip-this-step-if-you-already-have-an-engine).
```

Comment:
above it's mentioned that this guide is using gpt model but the link for engine creation you gave gives a command for llama? how do I find out what the right engine creation command is for gpt model?

Reply:
I think we are sticking with llama now.